cmd/admin-user-add.go

```diff
@@ -111,19 +120,34 @@ func mainAdminUserAdd(ctx *cli.Context) error {
 	// Get the alias parameter from cli
 	args := ctx.Args()
 	aliasedURL := args.Get(0)
+	accessKey := ""
+	secretKey := ""
+	policyName := ""
+	if len(args) == 1 {
+		accessKey = prompt.Input("Enter Access Key : ", completer,
+			prompt.OptionPrefixTextColor(prompt.Yellow))
+		secretKey = prompt.Input("Enter Secret Key : ", completer,
+			prompt.OptionPrefixTextColor(prompt.Yellow))
```

Comment:
The secret key **must** be read differently. PTAL at https://godoc.org/golang.org/x/crypto/ssh/terminal#Terminal.ReadPassword

Reply:
And yes thanks @aead we need to use prompt